### PR TITLE
Add Label to input components

### LINKED ISSUE #106: Add Label to input components
Add separate label and integrate into other Components that need a Label

### PR DESCRIPTION
**Pull request by stuarthendren**
Give multiple usage options and adds docs for them to storybbok.

Fixes #106

### CODE REVIEW

File: src/utils/ConditionalWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+/**
+ * Utility component for conditional wrapping.
+ */
+export const ConditionalWrapper: React.FC<{
+  condition: unknown
+  wrapper: (children: React.ReactElement) => React.ReactElement
```
Comment (jonnyelliot) [inline src/utils/ConditionalWrapper.tsx:8]: Could be `children: React.ReactChildren` to be more general

Comment (jonnyelliot) [reply src/utils/ConditionalWrapper.tsx:8]: The children prop could be optional.

**Review by jonnyelliot — APPROVED** [review]

File: src/utils/ConditionalWrapper.tsx (hunk 1 above)
Comment (stuarthendren) [reply src/utils/ConditionalWrapper.tsx:8]: I had `React.ReactNode` originally but the typescript complained about undefined. Will try `ReactChildren`.

Comment (stuarthendren) [reply src/utils/ConditionalWrapper.tsx:8]: I cant seem to get it to compile with ReactChildren. I guess if it return the unwrapped version you could get an array of react components and not a valid component. Like unwrapped in Fragment.

I've got it to work with ReactNode, but it causes problems downstream dealing with the undefined case. So I think the current version is the best we can do.

Maybe it shouldn't be a FC and would work with something different.

Ah, writing this comment has given me an idea...

Comment (stuarthendren) [reply src/utils/ConditionalWrapper.tsx]: Plot thickens, This version run the tests fine locally but they fail in the build.

**Pull request merged**